cmd/utils/flags.go

```diff
@@ -745,8 +754,11 @@ func setBootstrapNodesV5(ctx *cli.Context, cfg *p2p.Config) {
 // setListenAddress creates a TCP listening address string from set command
 // line flags.
 func setListenAddress(ctx *cli.Context, cfg *p2p.Config) {
-	if ctx.GlobalIsSet(ListenPortFlag.Name) {
+	switch {
+	case ctx.GlobalIsSet(ListenPortFlag.Name):
 		cfg.ListenAddr = fmt.Sprintf(":%d", ctx.GlobalInt(ListenPortFlag.Name))
+	case ctx.GlobalBool(MixFlag.Name):
+		cfg.ListenAddr = ":30313"
```

Comment:
geth sync port should not be hardcoded.  This is user-configurable, the user can choose any port arbitrarily, and it will continue to work properly with MIX Blockchain